Hotails/urls.py

```diff
@@ -19,5 +19,6 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', views.homepage, name='homepage')
+    path('', views.homepage, name='homepage'),
+    path('about/', views.about, name='about'),
 ]
```

Comment:
I think the URL of the about page should be recognized in `main/urls` and in `Hotails/urls` you should use the include option as @tamirmatok did in #62.
And in general, I think this PR depends on #62, that only if you logged in to the project you will be able to enter the about page.

Reply:
Good idea, I'll change that.

Reply:
It's your decision which pages should be available to guests and which not (personally I think including the about is ok).
my point was that the navbar should be available on all pages, with different content according to the user status (logged-in\ not logged-in)

Reply:
> It's your decision which pages should be available to guests and which not (personally I think including the about is ok). my point was that the navbar should be available on all pages, with different content according to the user status (logged-in\ not logged-in)

Yesterday Tamir made the changes in the login page, by adding an if statement for checking in the user is logged in or not, and presenting the navbar options as well. So it's all taken care of :)